README.md

```diff
@@ -1,18 +1,18 @@
 ## Service Tracker Example
 
-This is an example microservices application with a Javascript Web UI, a MongoDB database, and a series of API microservices. The idea is that various app developers would create Components for their corresponding apps. The overall config will add traits and allow the app to be fully deployed. 
-The application being deployed is shown in the following diagram: 
+This is an example microservices application with a Javascript Web UI, a MongoDB database, and a series of API microservices. The idea is that various app developers would create Components for their corresponding apps. The overall config will add traits and allow the app to be fully deployed.
+The application being deployed is shown in the following diagram:
 
 ![Application architecture diagram](service-tracker-diagram.jpg)
 
-> Full application original source here: https://github.com/chzbrgr71/service-tracker 
+> Full application original source here: https://github.com/chzbrgr71/service-tracker
 
 In this example, there are various roles that handle each aspect of the OAM application.
 
 * UI Developer
 * API Microservices Developer
 * MongoDB Admin
-* App Operator / SRE (handles applicaiton deployment in Kubernetes)
+* App Operator / SRE (handles application deployment in Kubernetes)
```

Comment:
That was the "fix" intended originally, but my editor also removed extra end of line space or tabs.
Do you want me to revert or are you ok with the extra changes?

Cheers,